### ISSUE TITLE
use all the features without a mouse

### ISSUE THREAD

**Issue by tess-phillips** [tess-phillips/ToDoList#6]

**Comment by tess-phillips** [comment]
e.g. when you write in the input and press enter nothing happens, but it should be as if add is pressed
